tests/e2e/specs/shopper/shopper-wc-blocks-checkout-failures.spec.js

```diff
@@ -15,7 +15,7 @@ import {
 	RUN_WC_BLOCKS_TESTS,
 } from '../../utils';
 
-const shippingDetails = config.get( 'addresses.customer.shipping' );
+const billingDetails = config.get( 'addresses.customer.billing' );
```

Comment:
Nitpick but maybe we could use constants above to get the identifiers with explicit names like `CUSTOME_BILLING_DETAILS` so that they're easier to identify and change. This might be a bigger thing to replicate across all instances but maybe we could just start with the files we are touching for now. WDYT?

Reply:
Not sure if I got this one, mind explaining it further? Like, exporting the value of `config.get( 'addresses.customer.shipping' )` from `../../utils` and importing it here, instead of retrieving them directly from the [config file](https://github.com/Automattic/woocommerce-payments/blob/develop/tests/e2e/config/test.json)?

Reply:
Ahh, apologies for the confusing wording. I was thinking something along the lines of :
```javascript
 const ADDRESS_CONFIG = config.get( 'addresses');
 const CARDS_CONFIG      = config.get( 'cards' );
 ...
 const billingDetails = ADDRESS_CONFIG['customer']['shipping'];
 const declinedCard = CARD_CONFIG['declined'];
```

The idea was to reduce the number of individual `config.get` calls to access the config values in different tests by having the root-level imports as constants and then using them. WDYT?

Reply:
Aha, got you now. Thanks for explaining it!

No prob on my end with adjusting it. But if you want another opinion, I think we better check about this with the ones who have been maintaining these tests for longer. I'm getting started with them.

I see we're directly using `config.get()` in many places and, in my opinion, that makes it clear at a glimpse where those values are coming from. I think that storing them in a variable and then using it would kind of obscure that. And I'm not sure how much of a value it'd provide. But that's just my 2c 🙂  I'd rather check with the devs that are more familiar with these tests.

Reply:
Since this is unrelated to the change at hand, resolving this conversation, can open a separate issue for this post discussing with Solaris folks if they feel it makes sense.